frontend/README.md

```diff
@@ -17,7 +19,7 @@ You will need the following things properly installed on your computer.
 
 To install the app, run the following commands in a command line:
 
-* `git clone https://github.com/fossasia/badgeyay this repository
+* git clone https://github.com/fossasia/badgeyay this repository
```

Comment:
Replace "git clone https://github.com/fossasia/badgeyay this repository" w/ `git clone https://github.com/fossasia/badgeyay`

Reply:
Done